src/secret-storage.ts

```diff
@@ -672,3 +672,30 @@ export function trimTrailingEquals(input: string): string {
         return input;
     }
 }
+
+/**
+ * Check whether the private keys exist in secret storage.
+ *
+ * @param secretStorage - The secret store using account data
+ * @returns map of key name to key info the secret is encrypted
+ *     with, or null if it is not present or not encrypted with a trusted
+ *     key
+ */
+export async function isStoredInSecretStorage(
```

Comment:
It's worth noting that the whole of the `secret-storage` module is public (https://pr3452--js-sdk-docs-previews.netlify.app/stable/modules/secretstorage) so putting this here exposes it as an external function to users of the js-sdk.

It's my opinion that the external API of the js-sdk is too cluttered as it is, and we should avoid exposing internal utility functions where we can. Can we find somewhere else for it?